vehicle/mkv/software/throttle/throttle.c

```diff
@@ -0,0 +1,90 @@
+#include "libs/adc/api.h"
+#include "libs/can/api.h"
+#include "libs/timer/api.h"
+#include <avr/interrupt.h>
+
+#include "throttle_config.h"
+#include "libs/gpio/api.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+/*
+ * Interrupts
+ */
+ volatile bool send_can = false;
+
+ void timer0_isr(void) {
+    send_can = true;
+}
+
+void pcint0_isr(void){
+	/*
+	   Check if shutdown Circuit nodes are open or closed. 
+	 */
+	// Check value of SS_IS, SS_ESTOP, and SS_BOTS pins
+
+}
+
+int main(void) {
+	// TODO: initalize motor controller by sending CAN msg
+    can_init(BAUD_500KBPS);
+    adc_init();
+    timer_init(&timer0_cfg);
+
+    gpio_set_mode(DEBUG_LED1, OUTPUT);
+    gpio_set_mode(DEBUG_LED2, OUTPUT);
+    gpio_set_mode(DEBUG_LED3, OUTPUT);
+    gpio_set_mode(LED1, OUTPUT);
+    gpio_set_mode(LED2, OUTPUT);
+    
+
+    gpio_set_mode(SS_ESTOP, INPUT);
+    gpio_set_mode(SS_IS, INPUT);
+    gpio_set_mode(SS_BOTS, INPUT);
+
+    gpio_enable_interrupt(SS_ESTOP);
+    gpio_enable_interrupt(SS_IS);
+    gpio_enable_interrupt(SS_BOTS);
+
+	// Initialize interrupts
+	sei();
+
+	// Wait for start up sequence
+	// 	completed precharge, brakes engaged, start button pressed
```

Comment:
I think we just need to listen for the "Ready to Drive" signal from the dashboard. But I will confirm.